README.md

```diff
@@ -14,8 +14,8 @@ This is a project of my responsive portfolio website that will hold my proffesio
 
 # Acknowledgment
 
-- for my conding partner
-- my mentor
+- to my conding partner
+- to my mentor
```

Comment:
Hi @mesfin345 Good job on the deployment of your project. I will approve it, please try to fix the conflict. Happy coding :+1:

Reply:
ok thanks